plugins/modules/aci_netflow_monitor_policy.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_netflow_monitor_policy
+short_description: Manage Netflow Monitor Policy (netflow:MonitorPol)
+description:
+- Manage Netflow Monitor Policies for tenants on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  netflow_monitor_policy:
+    description:
+    - The name of the Netflow Monitor Policy.
+    type: str
+    aliases: [ netflow_monitor, netflow_monitor_name, name ]
+  netflow_record_policy:
```

Comment:
Can this be removed? if so likely we should document that providing "" will remove the relationship.